website/src/components/Community.astro

```diff
@@ -0,0 +1,36 @@
+---
+import Contributors from "@src/components/Contributors.astro";
+import { Icon } from "@astrojs/starlight/components";
+---
+
+<div class="community">
+  <div class="community-grid"></div>
+  <div class="community-head">Community</div>
```

Comment:
The contrast ratio for this title and the description seems a little bit lower, you should keep some similarity to "Biome is fast" title